internal/region/region.go

```diff
@@ -37,6 +39,12 @@ var meta = []struct {
 		"https://api.eu-central-1.saucelabs.com",
 		"https://app.eu-central-1.saucelabs.com",
 	},
+	// APAC
+	{
+		"apac-southeast-1",
+		"https://api.apac-southeast-1.saucelabs.com",
+		"https://api.apac-southeast-1.saucelabs.com",
```

Comment:
Is this intentionally the same as the APIBaseURL? Or should it be `app.apac-southeast-1...`